### ISSUE TITLE
skipping loading seeds/fertilizer and continue with cultivator  

### ISSUE THREAD

**Issue by Johncodd99** [Courseplay/courseplay#6124]
im trying to spread slurry with the cultivator on the back but when the tanker is empty it just continues to harrow im using the auto drive to load it was working and now its not any ideas what iv done wrong ?

**Comment by Tensuko** [comment]
Since we have no Idea what Version you use and how is your setup and what are your settings we can't rly help.... logfile ?

**Comment by Johncodd99** [comment]
sorry the version is v6.02.00056

https://gist.github.com/Johncodd99/1a0c9f7f65834208ee71f7768832e594

**Comment by Tensuko** [comment]
> Always update to the latest build and check if the problem really still exists before reporting issues.

https://github.com/Courseplay/courseplay/wiki

https://github.com/Courseplay/courseplay/releases

**Comment by rooymans** [comment]
I'm having the same 'issue/experience' with the Kvernland DF-2 Seeder + Hopper. After it's empty, the worker just keeps going.

Kvernland DLC and Courseplay for FS19 v6.02.00064 on Hollandscheveld map (latest version)

[log.txt](https://github.com/Courseplay/courseplay/files/5282345/log.txt)

No warnings or error on the logfile. Clean modfolder and latest versions of all mods.

**Comment by schwiti6190** [comment]
@rooymans could you try this again, but with seeds selected in the hud 
![grafik](https://user-images.githubusercontent.com/58079399/94788636-25610f00-03d4-11eb-8f20-7ac5595be239.png)
still have to figure this one out.

**Comment by rooymans** [comment]
YEAH, that does 'fix' it. I hope that helps.

![Screenshot (24)](https://user-images.githubusercontent.com/56227140/94791919-de424080-040a-11eb-99b6-b910e436e71e.png)

**Comment by Tensuko** [comment]
Actually that is not 'fix' it is how it dos work... See Wiki https://github.com/Courseplay/courseplay/wiki/Mode-4-Changes
It was made so you can choose the filltype for sprayer like lime or fertilizer or for others manure or lime and so on, before it was a problem.... don't see an issue for this one then?

**Comment by VergylTantor** [comment]
I had this issue a few months ago on the Medicine Creek map where when the planter ran out of seed it continued on the course after it had stopped planting. It was repeatable on on that field of the map, but I can't remember the field number and it was prior to the Kvernland DF-2 release. As always I attempted to recreate the issue on a vanilla map and it did not repeat so I didn't post it as an issue.
I saw a YouTube video where the player had this issue on Field 2 of the Millennial Farmer map using a John Deere DB-60 planter mod.
I realize that this post is of little or no value other than to say that this appears to have happened with at least 3 different planters. It has also happened on the Medicine Creek, Millennial Farmer, Hollandscheveld and possibly the Eisersholt maps.
